charts/posthog/templates/_snippet-initContainers-wait-for-service-dependencies.tpl

```diff
@@ -0,0 +1,41 @@
+{{/* Common initContainers-wait-for-service-dependencies definition */}}
+{{- define "_snippet-initContainers-wait-for-service-dependencies" }}
+- name: wait-for-service-dependencies
+  image: busybox:1.34
+  command:
+    - /bin/sh
+    - -c
+    - >
+        {{ if .Values.clickhouse.enabled }}
```

Comment:
Maybe order them based on how fast they get ready usually (ClickHouse I suspect is last), so it feels like progress is happening and we get more detailed info from these logs.

Reply:
We don't control how resources are created so I don't think there's a real gain of reordering those. Also, if a service is up and running the `nc` calls takes milliseconds